regression-test/suites/nereids_rules_p0/mv/increment_create/left_anti_join_list_str_increment_create.groovy

```diff
@@ -0,0 +1,533 @@
+        sql"""
+        insert into orders_left_anti_1 values 
+        (2, 5, 'ok', 99.5, 'a', 'b', 1, 'yy', '2023-10-17'); 
+        """
+    }
+
+    // no window func + on partition col
+    def mv_sql_1 = """select l_shipdate, l_orderkey, count(l_shipdate), count(l_orderkey) 
+        from lineitem_left_anti_1
+        left anti join orders_left_anti_1
+        on lineitem_left_anti_1.l_shipdate = orders_left_anti_1.o_orderdate
+        group by l_shipdate, l_orderkey"""
+
+    def mv_sql_3 = """select l_shipdate, l_orderkey, l_partkey, l_suppkey  
+        from lineitem_left_anti_1
+        left anti join orders_left_anti_1
+        on lineitem_left_anti_1.l_shipdate = orders_left_anti_1.o_orderdate
+        """
+
+    // no window func + on not partition col
+    def mv_sql_4 = """select l_shipdate, l_orderkey, count(l_shipdate), count(l_orderkey)  
+        from lineitem_left_anti_1 
+        left anti join orders_left_anti_1 
+        on lineitem_left_anti_1.l_orderkey = orders_left_anti_1.o_orderkey 
+        group by l_shipdate, l_orderkey"""
+
+    def mv_sql_6 = """select l_shipdate, l_orderkey, l_partkey, l_suppkey  
+        from lineitem_left_anti_1
+        left anti join orders_left_anti_1
+        on lineitem_left_anti_1.l_orderkey = orders_left_anti_1.o_orderkey
+        """
+
+    // window func
+    def mv_sql_7 = """select l_shipdate, l_orderkey, l_partkey, 
+        count(lineitem_left_anti_1.l_shipdate) over (partition by lineitem_left_anti_1.L_SHIPDATE order by lineitem_left_anti_1.L_ORDERKEY) as window_count 
+        from lineitem_left_anti_1 
+        left anti join orders_left_anti_1 
+        on lineitem_left_anti_1.l_orderkey = orders_left_anti_1.o_orderkey 
+        group by l_shipdate, l_orderkey, l_partkey"""
+
+    def mv_sql_8 = """select l_shipdate, l_orderkey, l_partkey,
+        count(lineitem_left_anti_1.l_shipdate) over (partition by lineitem_left_anti_1.l_orderkey order by lineitem_left_anti_1.l_orderkey) as window_count
+        from lineitem_left_anti_1
+        left anti join orders_left_anti_1
+        on lineitem_left_anti_1.l_orderkey = orders_left_anti_1.o_orderkey
+        group by l_shipdate, l_orderkey, l_partkey"""
+
+    def mv_sql_9 = """select l_shipdate, l_orderkey, l_partkey, 
+        count(lineitem_left_anti_1.l_shipdate) over (order by lineitem_left_anti_1.l_orderkey) as window_count 
+        from lineitem_left_anti_1 
+        left anti join orders_left_anti_1 
+        on lineitem_left_anti_1.l_orderkey = orders_left_anti_1.o_orderkey 
+        group by l_shipdate, l_orderkey, l_partkey"""
+
+
+    def sql_all_list = [mv_sql_1, mv_sql_3, mv_sql_4, mv_sql_6, mv_sql_7, mv_sql_8, mv_sql_9]
+    def sql_partition_col_list = [mv_sql_1, mv_sql_3, mv_sql_7, mv_sql_9]
+    def sql_not_partition_col_list = [mv_sql_4, mv_sql_6]
+    def sql_increment_list = [mv_sql_1, mv_sql_3, mv_sql_4, mv_sql_6, mv_sql_8]
+    def sql_complete_list = []
+
+    // create mv with partition col
```

Comment:
the same as as below
comment should be` base on left table with part col` would be better

Reply:
Modified comment